docs/INSTALL-cloud.md

```diff
@@ -152,7 +170,7 @@ Do you want...
 
 - A Content Delivery Network to speed up worldwide access? [Configure a CDN](https://meta.discourse.org/t/enable-a-cdn-for-your-discourse/14857). We recommend [Fastly](http://www.fastly.com/).
 
-- Import old content from vBulletin, PHPbb, Vanilla, Drupal, BBPress, etc? [See our open source importers](https://github.com/discourse/discourse/tree/master/script/import_scripts).
+- Import old content from vBulletin, PHPbb, Vanilla, Drupal, BBPress, etc? [See our open source importers](https://github.com/discourse/discourse/tree/main/script/import_scripts).
```

Comment:
I don't want to keep adding things to this doc but maybe a word or two in this sentence could link to https://meta.discourse.org/t/how-to-migrate-from-one-platform-forum-to-discourse/197236?